internal/pipe/ko/ko.go

```diff
@@ -195,7 +199,7 @@ func (o *buildOptions) makeBuilder(ctx *context.Context) (*build.Caching, error)
 	case "go.version-m":
 		buildOptions = append(buildOptions, build.WithGoVersionSBOM())
 	case "none":
-		// don't do anything.
+		buildOptions = append(buildOptions, build.WithDisabledSBOM())
```

Comment:
The bug (unrelated to labels), let me know if you'd like it in a separate PR to release a hotfix

Reply:
if you don't mind, that would be awesome, yes!

thank you <3

Reply:
Here it is: #3861 :slightly_smiling_face: